glustercli/cmd/volume.go

```diff
@@ -89,6 +91,8 @@ func init() {
 
 	// Volume Expand
 	volumeExpandCmd.Flags().IntVarP(&flagExpandCmdReplicaCount, "replica", "", 0, "Replica Count")
+	volumeExpandCmd.Flags().IntVarP(&flagExpandCmdDistributeCount, "distribute", "", 0, "Distribute Count")
+	volumeExpandCmd.Flags().StringVarP(&flagExpandCmdSize, "size", "", "", "Size by which volume needs to be expanded.")
```

Comment:
need to change `StringVarP` to `StringVarP`

Reply:
Done